resize-observer/devicepixel2.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
+<link rel="match" href="devicepixel2-ref.html">
+<meta name="assert" content="Device pixel content box sizes and pixel snapping are correct in Resize Observer callback">
+<style>
+  html, body {
+    margin: 0;
+  }
+  #front {
+    position: absolute;
+    left: 0;
```

Comment:
Is there a reason we're bothering with `front` here at this point?  Can we just get rid of it and append to `body` instead, to simplify a bit more?

Reply:
so I just tried this and the bug does not repo in firefox if I append to body. I don't know if that still means the test should append to body or not. I honestly don't know the intent of the WPT. In WebGL conformance tests (which I'm very familiar with), we check in tests all the time that provoke bugs in individual browsers.

Reply:
> so I just tried this and the bug does not repo in firefox if I append to body

Thanks for testing that! Definitely want to be sure the bug still repro's in affected versions.

Can you just remove these #front styles? They should be unnecessary, and I confirmed I can still repro the bug in Firefox release with that change.  (Doesn't seem to make a difference.)

(I started to test appending to the body, but that's complicated by the fact that the we iterate over the elements using `children`, and the `<script>` element is one of the body's children right now. :))